.github/workflows/benchmark.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           - ubuntu-latest
           - windows-latest
     runs-on: ${{ matrix.runs-on }}
-    timeout-minutes: 10
+    timeout-minutes: 15
```

Comment:
It seems that 10min isn't enough on Windows based on the results on my fork: https://github.com/kou/csv/actions/runs/16061114221